src/parser/serializer.ts

```diff
@@ -1047,8 +1043,6 @@ export function serializeInto(
         index = serializeFunction(buffer, key, value, index, checkKeys, depth, serializeFunctions);
       } else if (value['_bsontype'] === 'Binary') {
         index = serializeBinary(buffer, key, value, index);
-      } else if (value['_bsontype'] === 'UUID') {
-        index = serializeBinary(buffer, key, value.toBinary(), index);
```

Comment:
I think changing `_bsontype` is a good thing, especially since UUID serialization/deserialization has not been released yet, so there's no need to worry about interoperability with existing bson package releases 👍